src/main/java/seedu/address/logic/parser/LogCommandParser.java

```diff
@@ -40,23 +48,32 @@ public LogCommand parse(String args) throws ParseException {
                             LogCommand.MESSAGE_ONE_FLAG));
                 }
                 String flag = argMultimap.getValue(PREFIX_FLAG).get();
-                assert(flag.equals(LogCommand.FLAG_RESET) || flag.equals(LogCommand.FLAG_OVERRIDE));
+                assert(flag.equals(LogCommand.FLAG_RESET) || flag.equals(LogCommand.FLAG_OVERRIDE)
```

Comment:
Could you include a Parse exception and error msg for unsupported flags? currently no error msg for 
`log n/central library -s`

Reply:
ok done!